vnext/ReactWindowsCore/packages.config

```diff
@@ -1,6 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="boost" version="1.68.0.0" targetFramework="native" />
-  <package id="ReactNative.Hermes.Windows" version="0.1.6" targetFramework="native" Condition="$(USE_HERMES)' == 'true'" />
-  <package id="ReactNative.V8Jsi.Windows" version="0.2.7" targetFramework="native" Condition="'$(USE_V8)' == 'true'" />
+  <!-- 
+    Enable alternate JS engines by un-commenting one of the following,
+    and setting either USE_V8 or USE_HERMES in React.Cpp.props 
+  <package id="ReactNative.Hermes.Windows" version="0.1.6" targetFramework="native" />
```

Comment:
Is there a better option you could recommend than uncommenting? We still want to support Hermes for build and I am working on the changes to make that happen. 
I was hoping to minimize the steps required to turn it on (ideally down to just setting USE_HERMES in one place).

Reply:
@Khalef1 unfortunately C++ projects only support packages.config, which do not have the ability to add conditionals or any msbuild related logic in them. 
.net projects can and should use `PackageReference` in their csproj files, which allows using Condition and any other msbuild constructs.